doc/source/dev/contributor/benchmarking.rst

```diff
@@ -136,6 +136,28 @@ To run only a single benchmark, such as ``KleeMinty`` from
 
    asv run --bench optimize_linprog.KleeMinty
 
+.. note::
+    Support for ``asv.conf.jsonc`` as a default config file path was added in
+    ``asv v0.6.4``. If developers must use an older version of ``asv``, they should
+    include ``--config=asv.conf.jsonc`` in their commands.
+
+.. note::
+
+    By default, the SciPy config uses ``mamba`` to manage ``asv`` environments,
+    which is preferable as explained in the ``asv`` docs:
```

Comment:
I might be a bit less aggressive on the recommendation here, though I don't really care much. `mamba`/`conda` may have other drawbacks depending on the computing environment you are using and your requirements, of course (i.e., speed isn't the only consideration, at least for me).